site/content/contribute/mobile/push-notifications/service.md

```diff
@@ -189,6 +189,10 @@ panic: Failed to load the apple pem cert err=failed to parse PKCS1 private key f
 
 This could also be because you generated a certificate for the wrong bundle ID. The bundle ID used in `mattermost-push-proxy.json` should be the same one as the app and the for the same app it was generated for.
```

Comment:
```suggestion
This could also be because you generated a certificate for the wrong bundle ID. The bundle ID used in `mattermost-push-proxy.json` should be the same one as the app, and should be for the same app it was generated for.
```